docs/berkeley-upgrade/migrating-archive-database-to-berkeley.mdx

```diff
@@ -109,42 +223,77 @@ Run the provided replayer app in migration mode:
 
    ```sh
-   cd /etc/mina/rosetta/archive
+   wget https://raw.githubusercontent.com/MinaProtocol/mina/berkeley/src/app/archive/zkapp_tables.sql
+
+   wget https://raw.githubusercontent.com/MinaProtocol/mina/berkeley/src/app/archive/create_schema.sql
 
    psql  -U postgres -c "CREATE DATABASE berkeley_migrated;"
 
    psql -U postgres -d berkeley_migrated -a -f create_schema.sql"
    ```
 
-4. Phase 1:
+4. Stage 1: Initial migration
+
+   4.a) Phase 1:
 
    ```sh
-   mina-berkeley_migration.exe -- --batch-size 2 --config-file /etc/mina/genesis_ledgers/mainnet.json --mainnet-archive-uri postgres://postgres:postgres@localhost/archive_balances_migrated --migrated-archive-uri postgres://postgres:postgres@localhost/berkeley_migrated--mainnet-blocks-bucket mina_network_block_data 
+   mina-berkeley-migration.exe --batch-size 2000 --config-file /etc/mina/genesis_ledgers/mainnet.json --mainnet-archive-uri postgres://postgres:postgres@localhost/archive_balances_migrated --migrated-archive-uri postgres://postgres:postgres@localhost/berkeley_migrated --mainnet-blocks-bucket mina_network_block_data --network mainnet
    ```
 
-4. Phase 2:
+   4.b) Phase 2:
 
    ```sh
    jq '.ledger.accounts' mainnet.json | jq  '{genesis_ledger: {accounts: .}}' > replayer_input_config.json"
 
-   mina-replayer -- --migration-mode --archive-uri postgres://postgres:postgres@localhost/ --input-file replayer_config_input.json --checkpoint-interval 100  --checkpoint-file-prefix migration
+   mina-replayer --migration-mode --archive-uri postgres://postgres:postgres@localhost/ --input-file replayer_config_input.json --checkpoint-interval 100  --checkpoint-file-prefix migration
    ```
 
+5. Stage 2: Initial migration
+
+   5.a) Phase 1:
+
+   ```sh
+   mina-berkeley-migration.exe --batch-size 2000 --config-file /etc/mina/genesis_ledgers/mainnet.json --mainnet-archive-uri postgres://postgres:postgres@localhost/archive_balances_migrated --migrated-archive-uri postgres://postgres:postgres@localhost/berkeley_migrated --mainnet-blocks-bucket mina_network_block_data --network mainnet
+   ```
+
+   5.b) Phase 2:
+
+   ```sh
+   mina-replayer --migration-mode --archive-uri postgres://postgres:postgres@localhost/ --input-file checkpoint-XXXX.json --checkpoint-interval 100  --checkpoint-file-prefix migration
+   ```
+
+5. Stage 2: Remainder migration
+
+   5.a) Phase 1:
+
+   ```sh
+   mina-berkeley-migration.exe --batch-size 2000 --config-file /etc/mina/genesis_ledgers/mainnet.json --mainnet-archive-uri postgres://postgres:postgres@localhost/archive_balances_migrated --migrated-archive-uri postgres://postgres:postgres@localhost/berkeley_migrated --mainnet-blocks-bucket mina_network_block_data --network mainnet --fork-state-hash "3NLdCBNrDseiDKvVj8rZ15k2oAUvx4XuCc8mzf6fL2CmqTJVVceM" 
+   ```
+
+   :warning: 3NLdCBNrDseiDKvVj8rZ15k2oAUvx4XuCc8mzf6fL2CmqTJVVceM - is some random hash, please do not user it on actual migration but learn from MF what is the fork point
+
+   5.b) Phase 2:
```

Comment:
@dkijania  I applaud this level of detail. After you get the technical content suggestions in and all content approved, I can help with the enumeration for the steps.